IntelFsp2WrapperPkg/IntelFsp2WrapperPkg.dec

```diff
@@ -137,3 +138,11 @@
     #1: measure FSP UPD region in one record (PCR1), the FSP code without UPD in another record (PCR0).
   #
   gIntelFsp2WrapperTokenSpaceGuid.PcdFspMeasurementConfig|0x00000000|UINT32|0x50000004
+  #
+  # Non-0 means PcdFspiUpdDataAddress will be ignored, otherwise PcdFspiUpdDataAddress will be used.
+  #
+  gIntelFsp2WrapperTokenSpaceGuid.PcdFspiUpdDataAddress|0x00000000|UINT32|0x50000005
+  #
+  # Non-0 means PcdFspiUpdDataAddress64 will be ignored, otherwise PcdFspiUpdDataAddress64 will be used.
+  #
+  gIntelFsp2WrapperTokenSpaceGuid.PcdFspiUpdDataAddress64|0x00000000|UINT64|0x50000006

```

Comment:
could we keep UINT64 PCD only and use it for both IA32 and X64 builds? that can simplify the implementation.

Reply:
It is better to keep consistent with FSP-T/M/S. And if need to change in the future, we could change FSP-T/M/S together.

Reply:
resolved